.travis.yml

```
@@ -50,7 +50,7 @@ addons:
 
 before_install:
   - echo "are changes related to source code?"
-  - bash $TRAVIS_BUILD_DIR/util/checkchanges.sh || test "${FORCE_NIGHTLY_UPLOAD}" = "yes" || travis_terminate 0;
+  - bash test "${FORCE_NIGHTLY_UPLOAD}" = "yes" || $TRAVIS_BUILD_DIR/util/checkchanges.sh || travis_terminate 0;
```

To avoid a build failure, this needs to be
```suggestion
  - test "${FORCE_NIGHTLY_UPLOAD}" = "yes" || bash $TRAVIS_BUILD_DIR/util/checkchanges.sh || travis_terminate 0;
```

---

Oops :flushed: